src/components/Rebuild/Rebuild.js

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { Button } from 'carbon-components-react';
+import Restart from '@carbon/icons-react/lib/restart/32';
+import { rebuildPipelineRun } from '../../api';
+import './Rebuild.scss';
+
+export class Rebuild extends Component {
+  handleRebuild = event => {
+    if (event) {
+      event.preventDefault();
+    }
+
+    /* eslint no-useless-rename: "error" */
+    const pipelinerunname = this.props.runName;
```

Comment:
why are these eslint comments needed?

Reply:
Destructuring occurs otherwise so if we were to run `npm run lint:fix`, the variable name would become runName instead. I really do want it to be `pipelinerunname` as this is to be used in the JSON payload for the actual rebuild function

I'll add this as a comment in the code to make it obvious for maintainers too

Reply:
Rather than renaming it here you should be able to do it where it's used in the payload below and avoid these eslint comments. Normally the comments are used to allow something that would otherwise be a violation, it's odd to see it used this way.

Reply:
I'd like to revisit these, but not a blocker for this PR.